gradle/libs.versions.toml

```diff
@@ -16,6 +16,7 @@ log4jApi = { strictly = '2.17.1' }
 slf4j = { strictly = '1.8.0-beta4' }
 nva-language = { strictly = '1.0' }
 awsSdk = { strictly = '1.12.129' }
+awsCdk = { strickly = '1.139.0' }
```

Comment:
Where is the CDK used? Also, "strickly"?

Reply:
It's not .....